src/src/PluginStructs/P078_data_struct.h

```diff
@@ -65,7 +65,8 @@ enum class SDM_MODEL {
   DDM18SD                = 3,
   SDM630                 = 4,
   SDM72_V2               = 5,
-  SDM320C                = 6
+  SDM320C                = 6,
+  TAC2100                = 7
```

Comment:
Adding a comma after `7` will avoid 2 lines changed when the _next_ model is added, and the compiler won't complain. (Also for the other lists like `constexpr p078_register_description` etc. Could have been added before, just wasn't done yet 🤔)